integration-test-common/src/test/java/com/datastrato/gravitino/integration/test/util/CloseContainerExtension.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.integration.test.util;
+
+import com.datastrato.gravitino.integration.test.container.ContainerSuite;
+import org.junit.jupiter.api.extension.BeforeAllCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CloseContainerExtension implements BeforeAllCallback {
```

Comment:
@xiaozcy 
Adding a comment about this class is preferred.

Reply:
Done